### PR TITLE
VIC slider bug fixes for IE

### PR DESCRIPTION
**Pull request by cehsu**
These changes fix the zoom slider bug in IE, and make the styles consistent across different browsers.

A bug related to fieldset width is also fixed.

win10_ie_11 0 1
win10_ie_11 0

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]
Verified locally. Looks great

File: src/js/vic-v2/components/PhotoField.jsx [hunk 1]
```diff
@@ -356,6 +356,23 @@ export default class PhotoField extends React.Component {
...
   }
 
+  onZoomMouseMove = (e) => {
+    if (this.state.mouseDown) {
+      this.onZoomSliderChange(e);
+    }
+  }
+
+  onZoomMouseDown = (e) => {
+    this.setState({ mouseDown: true });
```
Comment (jbalboni) [inline src/js/vic-v2/components/PhotoField.jsx]: I don't think you need to use `setState`, I think you can make it an instance property (`this.mouseDown = true`). Re-rendering on these events isn't necessary, as far as I can tell.

File: src/js/vic-v2/components/PhotoField.jsx [hunk 2]
```diff
@@ -665,6 +682,9 @@ export default class PhotoField extends React.Component {
                 aria-valuemin={this.state.minRatio}
                 aria-valuemax={this.state.maxRatio}
                 aria-valuenow={this.state.zoomValue}
+                onMouseDown={this.onZoomMouseDown}
+                onMouseUp={this.onZoomMouseUp}
+                onMouseMove={this.onZoomMouseMove}
                 onInput={this.onZoomSliderChange}/>
```
Comment (jbalboni) [inline src/js/vic-v2/components/PhotoField.jsx]: If this isn't firing in IE, does that mean that using the arrow keys to adjust the slider is also not working?

File: src/sass/modules/_m-schemaform.scss [hunk 3]
```diff
@@ -9,6 +9,12 @@ legend {
   padding: 0 0 .5em 0;
 }
 
+// Override IE
+fieldset {
```
Comment (jbalboni) [inline src/sass/modules/_m-schemaform.scss]: This is causing other pages in the app to have much smaller input boxes.

**Comment by jbalboni** [comment]
The slider looks oddly small in IE11 on Windows 8:

screen shot 2018-02-13 at 9 35 13 am

File: src/js/vic-v2/components/PhotoField.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/vic-v2/components/PhotoField.jsx]: Good catch, will replace onInput with onChange, which will capture keyboard input.

File: src/js/vic-v2/components/PhotoField.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/vic-v2/components/PhotoField.jsx]: Thanks, that's better, changed.

**Comment by cehsu** [comment]
@jbalboni good catch regarding the width issue, unset (unsupported by IE), has been replaced with auto.
screen shot 2018-02-13 at 10 25 26 am

File: src/sass/modules/_m-schemaform.scss (hunk 3 above)
Comment (cehsu) [reply src/sass/modules/_m-schemaform.scss]: Good catch, width has been set to 100%.
Large screen
bs_win10_ie_11 0 2
Small screen
bs_win10_ie_11 0 3

**Review by jbalboni — APPROVED** [review]

**Pull request merged**